pages/view/_iscnId.vue

```diff
@@ -421,27 +384,103 @@ export default class ViewIscnIdPage extends Vue {
     return `${ISCN_RAW_DATA_ENDPOINT}${this.iscnId}`
   }
 
+  created() {
+    const { iscnId } = this.$route.params
+    if (iscnId.startsWith(ISCN_PREFIX)) {
+      this.iscnId = iscnId
+    }
+  }
+
+  async mounted() {
+    this.txHash = await this.fetchTxHash()
+    if (!this.iscnId) {
+      const param = this.$route.params.iscnId
+      if (!isCosmosTransactionHash(param)) {
+        this.$nuxt.error({
+          statusCode: 400,
+          message: ErrorMessage.statusCode400,
+        })
+        return
+      }
+      const res = await this.fetchISCNByTx(param)
+      if (!res) {
+        this.$nuxt.error({
+          statusCode: 400,
+          message: ErrorMessage.statusCode400,
+        })
+        return
+      }
+      this.iscnId = res.records[0].id
+      this.$router.replace({ params: { iscnId: this.iscnId } })
+    }
+    if (!this.getISCNById(this.iscnId) || !this.owner) {
+      const res = await this.fetchISCNById(this.iscnId)
+      if (res) this.owner = res.owner
+    }
+    if (!this.getISCNById(this.iscnId)) {
+      this.$nuxt.error({ statusCode: 404, message: ErrorMessage.statusCode404 })
+    }
+    this.exifInfo = this.showExifInfo()
+  }
+
   async fetchTxHash() {
-    const res = await this.$axios.get(`${ISCN_TX_RAW_DATA_ENDPOINT}${this.iscnId}`)
+    const res = await this.$axios.get(
+      `${ISCN_TX_RAW_DATA_ENDPOINT}${this.iscnId}`,
+    )
     const txHash = this.getTxHash(res.data.txs)
     return txHash
   }
 
+  showExifInfo() {
+    const { exifInfo } = this.metadata
+    let exif
+    if (exifInfo) {
+      exif = {
+        Formate: exifInfo.Format,
+        Artist: exifInfo.Artist,
+        Camera:
+          exifInfo.Make && exifInfo.Model
+            ? `${exifInfo.Make} ${exifInfo.Model}`
+            : undefined,
+        Lens:
+          exifInfo.LensMake && exifInfo.LensModel
+            ? `${exifInfo.LensMake} ${exifInfo.LensModel}`
+            : undefined,
+        Exposure:
+          exifInfo.ExposureProgram &&
+          exifInfo.ExposureTime &&
+          exifInfo.FNumber &&
+          exifInfo.ISO
+            ? `${exifInfo.ExposureProgram}, 1/${(
+                1 / exifInfo.ExposureTime
+              ).toFixed(0)} s, f/${exifInfo.FNumber.toFixed(1)}, ISO ${
+                exifInfo.ISO
+              }`
+            : undefined,
+        Flash: exifInfo.Flash,
+        Date: exifInfo.CreateDate,
+        Size: exifInfo.Size,
+      }
```

Comment:
I think we should not use capitalized letter as object key, and should not assume this key will be directly display without i18n

Reply:
So I just need to switch the key to i18n (but still as the capitalized letter)? Or maybe we can split each item as the getter instead of using an object to render?